Documentation/direct-tools.md

```diff
@@ -16,7 +16,20 @@ If your pod dies, your mount will die with it.
 After you have created a pool as described in the [Block Storage](ceph-block.md) topic, you can create a block image and mount it directly in a pod.
 This example will show how the Ceph rbd volume can be mounted in the toolbox pod.
 
-After you have started and connected to the [Rook toolbox](ceph-toolbox.md), proceed with the following commands in the toolbox.
+### Start the Direct Mount pod
```

Comment:
One more thing... Now that I look at the doc again, how about if we make this a 2nd level header and move this section previous to the "Block Storage Tools" section. Then you can make a comment about how this direct mount pod is needed for both the block and file sections.

Reply:
Yes, I just did that, please check.